vllm/sequence.py

```diff
@@ -705,15 +706,11 @@ def __init__(
 
     @property
     def prompt(self) -> Optional[str]:
-        # All sequences in the group should have the same prompt.
-        # We use the prompt of an arbitrary sequence.
-        return self.seqs[0].prompt
```

Comment:
simplify code, `self.first_seq.prompt` is faster than `self.seqs[0]` .

more optimizations of this kind will come later.